CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur#1974](https://github.com/cyberark/conjur/issues/1974)
 - Correct unit tests and integration tests for audit, and correct a couple of issues found with them.
   [cyberark/conjur#1987](https://github.com/cyberark/conjur/issues/1987)
+- Aligning field to use ‘role’ under ‘subject'
```

Comment:
the change here is not clear. is it correct to say that this should be `Audit messages for password changes now have the field 'role' instead of 'user' under 'subject'`?